### ISSUE TITLE
Issues retrieving html for passing into metahtml

### ISSUE THREAD

**Issue by Tonnpo** [mikeizbicki/metahtml#157]
Hi MIke!

I am trying to apply metahtml to CLEANEVAL dataset. So, in each text files, the first line will have a format like this
```
<text id="http://www.emro.who.int/Publications/HealthEdReligion/AIDS/Chapter6.htm" title="Doing Away with Pretexts Justifying the Unlawful Behaviour" encoding="unset">
```
Then, I am able to extract the url which is the value of the attribute id as

http://www.emro.who.int/Publications/HealthEdReligion/AIDS/Chapter6.htm

However, when I look into the actual text file which is `397.html`, the title show
```
<title>Doing Away with Pretexts Justifying the Unlawful Behaviour</title>
```
If you follow the link, the title of this text file is the section 6 which is on page 18 of 36. Moreover, the link is in pdf format and so I am not able to download html from the URL. So, I am wondering how to retrieve information in html from pdf format? or is there any better way to work around on this?

Thanks very much!

**Comment by mikeizbicki** [comment]
I've downloaded the data and looked at the files.  When I look at the file `finalrun-input/397.html`, I see a proper html file.  You're right that when I visit the url http://www.emro.who.int/Publications/HealthEdReligion/AIDS/Chapter6.htm there is currently a pdf file at that url, but we don't care about that.  All of the webpages are going to have changed in the past 10 years since the dataset was released, so we can't be looking at what is currently on the web.  We should only be looking at the original html files that were originally processed to get the original ground truth.
